api/opds.py

```diff
@@ -1722,7 +1723,17 @@ def single_item_feed(
             )
 
         _db = Session.object_session(item)
-        work = license_pool.work or license_pool.presentation_edition.work
+
```

Comment:
Could we move the definition of `_db` below the definition of `work` here? If we fail to acquire a work, I don't think we'll need the db session.